.circleci/config.yml

```diff
@@ -150,6 +150,14 @@ jobs:
           name: Run integration tests for the Angular 6 application
           command: yarn integration:ng6
 
+  integration_ng7_tests:
```

Comment:
integration_ng7_tests - there is still a place moment where you need to specify for Circleci